internal/install/static_snapshot_yml.go

```diff
@@ -74,25 +74,41 @@ services:
       package-registry:
         condition: service_healthy
 
-  elastic-agent:
+  fleet-server:
     image: ${ELASTIC_AGENT_IMAGE_REF}
     depends_on:
       elasticsearch:
         condition: service_healthy
       kibana:
         condition: service_healthy
+    healthcheck:
+      test: "curl -f http://127.0.0.1:8220/api/status | grep HEALTHY 2>&1 >/dev/null"
+      retries: 60
+      interval: 1s
+    hostname: docker-fleet-server
+    environment:
+    - "FLEET_SERVER_ENABLE=1"
+    - "FLEET_SERVER_INSECURE_HTTP=1"
+    - "KIBANA_FLEET_SETUP=1"
+    - "KIBANA_FLEET_HOST=http://kibana:5601"
```

Comment:
How does the fleet-server know username / password for Kibana?

Reply:
It's standard elastic/changeme